### PR TITLE
Updated pymc.Exponential docstring

### PR DESCRIPTION
**Pull request by kimbochen**
- Added a space before the colon in the Parameter section.
- Put the parameter name `lam` in back ticks.
+ [ ] right before it's ready to merge, mention the PR in the RELEASE-NOTES.md

### CODE REVIEW

File: pymc/distributions/continuous.py [hunk 1]
```diff
@@ -1461,8 +1461,8 @@ class Exponential(PositiveContinuous):
 
     Parameters
     ----------
-    lam: float
-        Rate or inverse scale (lam > 0)
+    lam : float
```
Comment (OriolAbril) [inline pymc/distributions/continuous.py]:
```suggestion
    lam : tensor_like of float
```

File: pymc/distributions/continuous.py [hunk 2]
```diff
@@ -1461,8 +1461,8 @@ class Exponential(PositiveContinuous):
...
+        Rate or inverse scale (`lam` > 0)
```
Comment (OriolAbril) [inline pymc/distributions/continuous.py]:
```suggestion
        Rate or inverse scale (``lam`` > 0)
```

dosctrings are in rST format which needs two backticks for inline code formatting.

**Review by OriolAbril — COMMENTED** [review]
Can you also add the close-fig context to the plot directive as show in https://pymc-data-umbrella.xyz/en/latest/sprint/docstring_tutorial.html#section-independent-comments?

**Comment by kimbochen** [comment]
Thanks for reviewing. I'll look into the docs and fix it.

**Review by OriolAbril — APPROVED** [review]

**Pull request merged**